app/views/contexts/summary/group.py

```diff
@@ -77,10 +92,50 @@ def _build_blocks(
                     ]
                 )
 
-        return blocks
+            elif block["type"] in ["ListCollector"]:
+                list_collector_block = ListCollectorBlock(
+                    routing_path=routing_path,
+                    answer_store=answer_store,
+                    list_store=list_store,
+                    progress_store=progress_store,
+                    metadata=metadata,
+                    response_metadata=response_metadata,
+                    schema=schema,
+                    location=location,
+                    language=language,
+                )
+
+                section = schema.get_section(location.section_id)
+                summary = section.get("summary", {})
+                if summary_items := summary.get("items"):
+                    for summary_item in summary_items:
+                        if (
+                            summary_item["type"] == "List"
+                            and summary_item["for_list"] == block["for_list"]
+                        ):
+
+                            list_summary_element = (
+                                list_collector_block.list_summary_element(summary_item)
+                            )
+                            blocks.extend([list_summary_element])
+                            links["add_link"] = SummaryLink(
+                                text=list_summary_element["add_link_text"],
+                                url=list_summary_element["add_link"],
+                                attributes={"data-qa": "add-item-link"},
+                            )
+                            links["empty_list_text"] = list_summary_element[
```

Comment:
This isn't a link. It should be at the group level. I would initialise links and placeholder_text as part of group, then set it when building the block. Instead of the method returning multiple.

Reply:
Ok, added this on group level (but giving values when going through blocks), was I supposed to rename the var `placeholder_text` instead of `empty_list_text`?